zipkin-server/README.md

```diff
@@ -268,6 +268,16 @@ The following apply when `STORAGE_TYPE` is set to `elasticsearch`:
                            to 0 as it would mean a machine failure results in data loss.
     * `ES_USERNAME` and `ES_PASSWORD`: Elasticsearch basic authentication, which defaults to empty string.
                                        Use when X-Pack security (formerly Shield) is in place.
+    * `ES_CREDENTIALS_FILE_PATH`: Elasticsearch credentails properties file path, which defaults to empty
+                               string. The suffix of file should be `.properties`. The keys of
```

Comment:
is the requirement for a `.properties` file suffix a Spring restriction ? If not, any special reason to insist on it ?

Reply:
It's not a Spring restriction, but a java Properties file instead. I think it's a conventional way to append ".properties" to the file name. It prompts the format of the file.

Reply:
I feel we should rather not enforce the extension of the credentials file, sites may have naming conventions of their own for such files.  @anuraaga any opinion here ?

Reply:
done.